packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Wysiwyg/Editor.tsx

```diff
@@ -60,12 +76,55 @@ const Editor = React.forwardRef<EditorApi, EditorProps>(
         inputStyle: 'contenteditable',
       });
 
-      // @ts-expect-error – doesn't think command exists?
+      // Register custom commands for markdown formatting
       CodeMirror.commands.newlineAndIndentContinueMarkdownList =
         newlineAndIndentContinueMarkdownList;
-      editorRef.current.on('change', (doc) => {
+
+      CodeMirror.commands.bold = () => markdownHandler(editorRef, 'Bold');
+      CodeMirror.commands.italic = () => markdownHandler(editorRef, 'Italic');
+      CodeMirror.commands.underline = () => markdownHandler(editorRef, 'Underline');
+      CodeMirror.commands.link = () => markdownHandler(editorRef, 'Link');
+
+      const changeHandler = (doc: CodeMirror.Editor) => {
         onChangeRef.current(name, doc.getValue());
-      });
+      };
+
+      editorRef.current.on('change', changeHandler);
+
+      // Handle paste event to detect images
+      const handlePaste = (_cm: CodeMirror.Editor, event: ClipboardEvent) => {
```

Comment:
could you rename _cm to something like `editor`? we try to have explicit variable names, and CM in strapi generally refers to the content manager